qp_klp/tests/test_klp.py

```diff
@@ -345,8 +345,10 @@ def test_sequence_processing_pipeline(self):
         # NOTE: we are not creating a new job for this test, which is fine
 
         # fix self.sample_csv_data
+        # this should also confirm that the leading zero in '0640193' is
+        # silently removed in the prep file results.
         self.sample_csv_data[21] = self.sample_csv_data[21].replace(
-            'SKB8.640193XX', 'SKB8.640193')
+            'SKB8.640193XX', 'SKB8.0640193')
```

Comment:
I _think_ the change should be:
```suggestion
            'SKB8.640193XX', '0SKB8.640193')
```